Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "partywithray-proof-of-membership"
+version = "0.1.0"
+edition = "2021"
```

Comment:
2022 ?

Reply:
This is the Rust edition.  2015,2018, or 2021.  2021 is the latest.